service/role.go

```diff
@@ -339,6 +364,19 @@ func (r *roleService) updateRoleToken(ctx context.Context, domain, role, proxyFo
 	return rt.(*RoleToken), err
 }
 
+// TODO:
+func roleCacheMemoryUsage(data *cacheData) int64 {
+	tokenSize := int64(unsafe.Sizeof(data.token)) + int64(len(data.token.Token))
+	expiryTimeSize := int64(unsafe.Sizeof(data.token.ExpiryTime)) + int64(len(strconv.FormatInt(data.token.ExpiryTime, 10)))
```

Comment:
1. point to struct
2. struct size (string struct + int64)
3. string length

```suggestion
	tokenSize := int64(unsafe.Sizeof(data.token)) + int64(unsafe.Sizeof(*(data.token))) + int64(len(data.token.Token))
```
(not sure working or not)

Reply:
@WindzCUHK fixed in https://github.com/AthenZ/athenz-client-sidecar/pull/18/commits/8363817895e6a2447ba029bf48de8dcf8e404d52